src/platform/site-wide/sass/modules/_m-footer.scss

```diff
@@ -5,6 +5,7 @@
   background: $color-primary-darkest;
   position: relative;
   overflow: hidden;
+  box-shadow: 0 50vh 0 50vh $color-primary-darkest;
```

Comment:
is the `50vh` a specific number based off other properties or just eyeballing it? just curious

Reply:
50% of the viewport height in both directions means it'll always cover everything.

... :thinking: Unless, perhaps, the bottom of the footer is above halfway down the screen. Could do `100vh`, now that you mention it. :man_shrugging:

Reply:
Actually, I take it back. After fiddling with it for a bit, I remember why this works again (had to look up how `box-shadow` worked again):

```sass
box-shadow: 0 50vh 0 50vh $color-primary-darkest;
```
`offset-x`: 0
`offset-y`: 50% of the viewport height
`blur-radius`: 0 (don't blur)
`spread-radius`: 50% of the viewport height
`color`: same as the footer

The reason this works is because of the spread radius. Essentially, we're shifting the box shadow down half a screen, then stretching it in every direction the same amount (half the viewport height), so the top of the shadow lines up exactly with the top of the footer.

What this _also_ means is that the shadow spreads _down_ half a screen, so even if the footer was at the very top of the screen, the shadow would start in the middle (50vh down) and spread out 100vh, covering the entire screen.

Reply:
this was as much an inquiry into the units as it was a needed refresher on how `box-shadow` works, so thank you 😉